### ISSUE TITLE
Create an Interactive Map feature that shows the heritage locations of our collective community 

### ISSUE THREAD

**Issue by EllaCodes4** [Latina-Dev/latina-dev#101]
## Describe the solution you'd like

Isis proposed we include an interactive map feature on our About Page (or possibly members page?) where we show a world map that places a pin on the heritage countries that LatinaDev members collectively represent (Peru, Mexico, Brazil, etc.)

## Additional context

Isis is currently developing the About Page

**Comment by FrancesCoronel** [comment]
Anel expressed interest in tackling this issue

**Comment by EllaCodes4** [comment]
Great, thank you Anel!

**Comment by EllaCodes4** [comment]
Hi Anel!

Happy New Year! Origin/heritage countries are optional for members but if
available, can be found on the member cards! Current location is not
available right now.

Thanks for your contribution!

**Comment by EllaCodes4** [comment]
Great job with this project, @anelguel! 👏🏽 Thanks for your contribution!

Can we add the country’s flag to the title? For example when you drop the
pin, it’d read: “Heritage location: Brazil 🇧🇷 “?

**Comment by anelguel** [comment]
Yeah, I can see if I if there is support for emoji's in the mapping platform I'm using. 
Also, let me know if/what title & description text you'd all like added.
More soon.

**Comment by anelguel** [comment]
I found out that there isn't support for flag emojis. Instead, I added flag pictures to the pop-ups.

Demo:
![Image](https://github.com/user-attachments/assets/884698dc-3e19-458d-89a4-d9ec0099a8b1) The default

![Image](https://github.com/user-attachments/assets/28269564-00c1-49ee-81fb-6b4337acfdea) Panning/zooming in/clicking the drop pin

![Image](https://github.com/user-attachments/assets/276e86b3-55b2-4269-a11c-080b4ad8f6d3) Scrolling to the end of the pop-up, where you'll see the country flag.

**Comment by EllaCodes4** [comment]
Hi @anelguel! Wow, this looks so good! Thanks again for this amazing
contribution!!! 👏🏽 Teamwork makes the dream work! 🙌🏽

**Comment by anelguel** [comment]
Sure thing, I just created a PR 🤠
